cms/envs/common.py

```diff
@@ -1795,6 +1808,7 @@
 
     # alternative swagger generator for CMS API
     'drf_spectacular',
+    'openedx_events'
```

Comment:
[nit] trailing comma
```suggestion
    'openedx_events',
```

Reply:
[fixed]